lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -41,11 +41,22 @@ def self.run(params)
           Actions.sh "echo y | android update sdk --no-ui --all --filter tools"
         end
 
+        # Accept licenses for all available packages
+        FastlaneCore::CommandExecutor.execute(command: "echo y | #{sdk_manager} --licenses",
```

Comment:
The `echo y |` accepts the licenses.

Can you let me know, the usecase behind this?

Reply:
I didn't see that happening... Perhaps it was my installation? Or a change in the android sdk?

Reply:
I tested it on my system

```
 ❯ sdkmanager --licenses

3 of 6 SDK package licenses not accepted. 100% Computing updates...
Review licenses that have not been accepted (y/N)?
...
Accept? (y/N): 3 licenses not accepted
```

It is: 

- mips-android-sysimage-license
- google-gdk-license
- android-googletv-license

The missing license of those packages didn't have any negative impact on my system.

And `echo y | sdkmanager --licenses` did not accept those licenses. Does it for you?

Reply:
Ah – I know what I did. I used `yes` rather than `echo yes` which sends a continuous stream of `y`s rather than just one.

This is needed for accepting the licenses as there are multiple ones to accept.

When I ran it without the separate license acceptance lines (i.e. just with `command: "yes | #{sdk_manager} '#{packages.join("' '")}'"`) then the licenses were *not* accepted.

This is on an Amazon Linux AMI instance (2017.09)